Authentication/Validators/TextInputValidatorType.swift

```diff
@@ -15,6 +15,10 @@ public enum ValidationResult {
         return .Invalid(errors: [errorMessage])
     }
     
+    public static func valid() -> ValidationResult {
```

Comment:
I would like it to stay a function just to be consistent with the invalid function. But maybe it's just a whim of mine.